scripts/run_test

```diff
@@ -13,7 +13,7 @@ docker run --rm -v $VOLUME:/beebop/storage mrcide/beebop-py:$API_BRANCH \
        ./scripts/download_db --small storage
 docker network create $NETWORK > /dev/null || /bin/true
 
-docker run -d --rm --name $NAME_REDIS --network=$NETWORK redis:5.0
+docker run -d --rm --name $NAME_REDIS --network=$NETWORK -p 6379:6379 redis:5.0
```

Comment:
Exposing the redis port here so we can access it from the app when running outside docker.